src/presolve.jl

```diff
@@ -239,6 +245,8 @@ function extract_reduced_problem!(ps::PresolveData{T}) where {T}
 
         # Set new column
         pb.acols[jnew] = Col{T}(cind, cval)
+
+        # TODO: Could we just use ps.var_types[jold] instead?
```

Comment:
No, because we may change variable types during presolve.
For instance, a `GENERAL_INTEGER` may be switched to `BINARY` if we realize its bounds are `[0, 1]`.
Some continuous variables may also be flagged as integer (the so-called "impled-integer" variables).